targets/BMS/main.cpp

```diff
@@ -159,7 +158,7 @@ int main() {
     canStackDriver.Timer = &timerDriver;
     canStackDriver.Nvm = &nvmDriver;
 
-    CO_NODE_SPEC canSpec = {
+    CO_NODE_SPEC canSpecOne = {
```

Comment:
Why did you rename this?

Reply:
At one point the idea of this branch was to research having more than one node. I believe this change is residual from when it was handed to me. There was a canSpecOne-canSpecThree. This is no longer needed as we found that to be an unnecessary change. I will change it bac to canSpec along with the name of the node to canNode (It is currently canNodeOne).